Makefile

```diff
@@ -33,11 +33,11 @@ test-existing: generate fmt vet manifests
 	rm -f cover.out cover.out.tmp cover.json
 
 # Build manager binary
-manager: generate fmt vet
+manager: generate fmt checks
 	go build -o bin/manager main.go
 
 # Run against the configured Kubernetes cluster in ~/.kube/config
-run: generate fmt vet
+run: generate fmt checks
```

Comment:
Is it possible to have `generate fmt vet lint` instead of `generate fmt checks`?

Reply:
As mentioned above, golangci-lint includes go vet. Let me know if you still want go vet added back

Reply:
Re-added as per discussion